src/test/java/plugins/MavenPluginTest.java

```diff
@@ -246,4 +248,20 @@ private void assertHasModule(MavenModuleSet job, String name) {
         find(by.xpath("//a[@href='%s/']", name)).click();
         assertThat(driver.getCurrentUrl(), equalTo(build.module(name).url.toExternalForm()));
     }
+
+    @Issue("JENKINS-22252")
+    @WithPlugins({"maven-plugin@2.12", "tasks"})
```

Comment:
Well, reading the code again... I think my initial question still makes sense :smile: